lib/new_relixir/agent.ex

```diff
@@ -127,7 +139,7 @@ defmodule NewRelixir.Agent do
     ]
     base_url = String.replace(@base_url, "~s", host)
     segments = List.flatten([base_url, urljoin(args ++ base_args)])
-    Enum.join(segments) |> String.to_char_list
+    Enum.join(segments)
   end
 
   defp urljoin([]), do: []
```

Comment:
I wonder if we should use something like [encode_query](https://hexdocs.pm/elixir/URI.html#encode_query/1) here 🤔